validation/src/test/scala/hmda/validation/rules/lar/macro/MacroSpec.scala

```diff
@@ -27,11 +27,32 @@ abstract class MacroSpec extends SummaryEditCheckSpec with BeforeAndAfterAll wit
     system.terminate()
   }
 
-  protected def newLarSource(lars: List[LoanApplicationRegister], numOfRelevantLars: Int, relevantDef: LoanApplicationRegister => LoanApplicationRegister, irrelevantDef: LoanApplicationRegister => LoanApplicationRegister) = {
-    val relevantLars = lars.map(lar => relevantDef(lar)).take(numOfRelevantLars)
-    val irrelevantLars = lars.map(lar => irrelevantDef(lar)).drop(numOfRelevantLars)
+  protected def newLarSource(
```

Comment:
I've refactored this to eliminate the extra mappings. I believe that should address this!